packages/dev/loaders/src/SPLAT/splatFileLoader.ts

```diff
@@ -83,6 +83,19 @@ export class SPLATFileLoader implements ISceneLoaderPluginAsync, ISceneLoaderPlu
         return new SPLATFileLoader(options[SPLATFileLoaderMetadata.name]);
     }
 
+    /**
+     * set option flipY
+     */
+    public set flipY(flipY: boolean) {
+        this._loadingOptions.flipY = flipY;
+    }
+
+    /**
+     * get option flipY
+     */
+    public get flipY(): boolean {
+        return !!this._loadingOptions.flipY;
+    }

```

Comment:
Seems weird to add these since we don't have it for `keepInRam`, and we are trying to move away from mutating the plugin instance and instead passing through options. Personally I would not add these, and instead just expect users to use the newer APIs, e.g.
```ts
LoadAssetContainerAsync(splatUrl, scene, {
  flipY: true,
});
```

Reply:
Makes sense. Fixed.